.gitignore

```diff
@@ -15,3 +15,4 @@ adminer.php
 /nbproject
 /.php_cs.cache
 /doc
+.history
```

Comment:
@tridnguyen thanks for the PR, man! It looks good.

Why did you add this to the .gitignore file?